src/test/java/RedisPoolTest.java

```diff
@@ -0,0 +1,43 @@
+import org.junit.Test;
+import vi.viredis.client.ViRedis;
+
+public class RedisPoolTest {
+    private RedisPool redisPool;
+    private String host = "127.0.0.1";
+    private int port = 6379;
+
+    @Test
+    public void testCreateRedisConnectionPool() {
+        redisPool = new RedisPool(10, host, port);
+        assert redisPool.size() == 10;
```

Comment:
In JUnit don't use the java assert, use the assertEquals() methods.

Reply:
done!